lib/pavilion/builder.py

```diff
@@ -230,6 +232,21 @@ def exists(self):
         """Return True if the given build exists."""
         return self.path.exists()
 
+    def log_updated(self) -> Union[float, None]:
+        """Return the last time the build log was updated."""
+
+        if self.tmp_log_path.exists():
+            try:
+                return self.tmp_log_path.stat().st_mtime
+            except OSError:
+                pass
+
```

Comment:
Under what circumstance might we trigger an OS error here? Why do we pass if we encounter one?